thread-discussions.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Thread Discussions'
+---
+
+<Frame>![Example Thread Discussion](/public/images/thread-discussion.png)</Frame>
+
+Thread discussions allow you to lead side conversations for any thread with your wider team.
```

Comment:
I think we should try and keep the language a bit simpler for the docs so that it's less indirect/formal:

```suggestion
Thread discussions allow you to discuss support requests with your team in Slack.
```